src/config.ts

```diff
@@ -39,14 +49,31 @@ const defaultConfig: Config = {
   setupProvider: async baseProvider => baseProvider,
 
   coverage: false,
+
+  node: {
+    allowUnlimitedContractSize: false,
+    gasLimit: DEFAULT_BLOCK_GAS_LIMIT,
+    gasPrice: 20e9, // 20 gigawei
+  },
 };
 
 function getConfig(): Config {
   const location = findUp.sync(CONFIG_FILE, { type: 'file' });
   const providedConfig: Partial<Config> = location !== undefined && fs.existsSync(location) ? require(location) : {};
 
+  if (providedConfig.blockGasLimit !== undefined) {
+    log(`blockGasLimit is deprecated. Use node.gasLimit instead. See ${configHelpUrl} for details.`);
```

Comment:
I'd use [`util.deprecate`](https://nodejs.org/api/util.html#util_util_deprecate_fn_msg_code) instead.

Reply:
`util.deprecate` is for function calls and not every `getConfig` call has deprecated fields. How we make it conditional? You suggest to double wrap `getConfig` in conditional `util.deprecate`?

Reply:
Ah you're right, I got confused with how we used it in test-helpers. Yes, it might be confusing to use it then.

I'd consider though doing something like [this](https://github.com/OpenZeppelin/openzeppelin-test-helpers/blob/master/src/expectRevert.js#L37) so that users have a clearer idea of where the error message comes from: given the number of moving pieces involved in testing the message might be a bit cryptic (though the link to the docs is a great idea!).